.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: go
+go:
+  - "1.12"
```

Comment:
1.13 is ready.

Reply:
I've added TODO to handle migration in a dedicated PR. I have this GO111MODULE=on env spread in few places and I need more testing to make sure that upgrade does not break things.